irc_channel.c

```diff
@@ -625,6 +625,24 @@ static gboolean control_channel_invite( irc_channel_t *ic, irc_user_t *iu )
 	return TRUE;
 }
 
+static void control_channel_kick( irc_channel_t *ic, irc_user_t *iu )
+{
+	struct irc_control_channel *icc = ic->data;
+	bee_user_t *bu = iu->bu;
+	
+	if( bu == NULL )
+		return;
+	
+	if( icc->type != IRC_CC_TYPE_GROUP )
+	{
+		irc_send_num( ic->irc, 482, "%s :Kicks are only possible to fill_by=group channels", ic->name );
```

Comment:
Hrm, anyway. remove_buddy() takes a group param yeah, but if NULL it should remove the buddy altogether. That arg is inherited from Gaim really, I think. And since BitlBee doesn't support multiple groups per contact (because indeed mostly who cares about groups), it's effectively all the same.